generate.sh

```diff
@@ -2,7 +2,20 @@
 
 srcdir=$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )
 
+function ask1() { local yn; echo -n "$* [Y/n] "; read -r -n 1 yn; [[ "$yn" =~ [Yy] ]] && echo && return 0; [ -z "$yn" ] && return 0; echo; return 1; }
+
 for f in "$srcdir"/{,.}*.mustache; do
-    echo "Generating $(basename "$f" .mustache)..."
-    mustache meta.yml "$f" > "$(basename "$f" .mustache)"
+    target=$(basename "$f" .mustache)
+    case "$target" in
+        *.opam)
+            shortname=$(grep -e "shortname:" meta.yml | sed -e "s/^shortname: //")
+            target=${target/coq/coq-$shortname}
+            ;;
```

Comment:
This doesn't cover the case of `coq-extracted` templates.

Reply:
@Zimmi48 indeed: IINM the final name of this file should be `{{extracted_shortname}}.opam`…

Reply:
Right